### PR TITLE
[ENG-5318] Add is-mobile helper

### PR DESCRIPTION
**Pull request by futa-ikeda**
-   Ticket: [ENG-5318] 
-   Feature flag: n/a

## Purpose
- Add a new helper to avoid having to make unneeded component.ts and controller.ts files

## Summary of Changes
- Create new `is-mobile` helper
- Test for new helper
- *Not currently implemented in actual pages*

## Screenshot(s)
- NA

## Side Effects
- None

## QA Notes
- No QA needed as nothing functional is changing

[ENG-5318]: https://openscience.atlassian.net/browse/ENG-5318?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/integration/helpers/is-mobile-test.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+module('Integration | Helper | is-mobile', function(hooks) {
+    setupRenderingTest(hooks);
+
+    // Replace this with your real tests.
+    test('it renders', async function(assert) {
+        setBreakpoint('mobile');
+
+        await render(hbs`
+<div data-test-div>{{if (is-mobile) 'mobile' 'not-mobile'}}</div>`);
```
Comment (cslzchen) [inline tests/integration/helpers/is-mobile-test.ts:16]: Question: the style looks off for me but I am not familiar either. Maybe something like this?

```javascript
await render(hbs`
        <div data-test-div>{{if (is-mobile) 'mobile' 'not-mobile'}}</div>
`);
```

**Comment by futa-ikeda** [comment]
Hmm, this actually seems redundant based on what is in the [ember-responsive docs](https://github.com/freshbooks/ember-responsive?tab=readme-ov-file#getting-started)
```
{{#if (media 'isDesktop')}}
  Desktop view!
{{/if}}
```

File: tests/integration/helpers/is-mobile-test.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+import { setBreakpoint } from 'ember-responsive/test-support';
+
+module('Integration | Helper | is-mobile', function(hooks) {
+    setupRenderingTest(hooks);
+
+    // Replace this with your real tests.
```
Comment (cslzchen) [inline tests/integration/helpers/is-mobile-test.ts:11]: NotABlocker: Remove this comment?

**Review by cslzchen — APPROVED** [review]
LGTM, with a couple of comments/questions

File: tests/integration/helpers/is-mobile-test.ts (hunk 1 above)
Comment (futa-ikeda) [reply tests/integration/helpers/is-mobile-test.ts:16]: Yea, our linter doesn't like or know how to handle the `render(hbs` we use here. It always gives an "Incorrect indentation" message and says the indentation should be 0, for whatever reason. I like how your suggestion is styled much more, so I will disable the linter for this bit and update the indentation level

**Pull request merged**